SETUP.md

```diff
@@ -35,10 +35,16 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
     </details>
 
 1. `rake install`
+  * If you needed `bundle exec` in front of the previous command, you will need it here too.
```

Comment:
I would consider just putting `bundle exec` in front of the rake commands throughout this doc, instead of including the warnings - it's safe to do even if you don't _have_ to, and it's fairly common to need this to ensure the correct gems are used.

Reply:
I was wondering about that - I will make that change.